### DISCUSSION TITLE
GOOGLE_ANALYTICS_MEASUREMENT_ID能否设置多个值？

### DISCUSSION THREAD

**Discussion by 0verleaves** [lobehub/lobehub#2060] [General Question] [answered]
我在vercel部署了lobechat，并且绑定了多个域名，现在我想对这些网站都进行数据分析。想询问一下Google Analytics能不能使用逗号分隔的形式绑定多个Measurement ID？
![Screenshot_2024-04-16-11-19-32-73_a252b927494330cdc2c8ba3b3f952e5e.jpg](https://github.com/lobehub/lobe-chat/assets/54617500/a56ae6dd-cfc9-40b1-b345-0539cb7c691d)

**Comment by arvinxx** [comment]
不计划支持，你自己开多个 project 不行吗？

**Comment by 0verleaves** [comment]
> 不计划支持，你自己开多个 project 不行吗？

github似乎不能fork两个相同的仓库，而且开多个 project 不方便管理啊

**Reply by arvinxx** [reply]
一个github 项目可以创建多个vercel项目的

**Reply by arvinxx** [reply, marked as answer]
> 而且开多个 project 不方便管理啊

既然有需求欢迎来贡献代码。我没这个需求所以我不会加，如果其他人有这个需求也可以让其他人来加

**Reply by 0verleaves** [reply]
好的，谢谢解答
